sdk/core/Azure.Core/src/Shared/OperationInternalOfT.cs

```diff
@@ -99,26 +105,54 @@ internal class OperationInternal<T> : OperationInternalBase
             string? operationTypeName = null,
             IEnumerable<KeyValuePair<string, string>>? scopeAttributes = null,
             DelayStrategy? fallbackStrategy = null)
-            : base(clientDiagnostics, operationTypeName ?? operation.GetType().Name, scopeAttributes, fallbackStrategy)
         {
+            _diagnostics = clientDiagnostics;
             _operation = operation;
             _rawResponse = rawResponse;
             _stateLock = new AsyncLockWithValue<OperationState<T>>();
+            _responseLock = new AsyncLockWithValue<Response>();
+            _fallbackStrategy = fallbackStrategy;
+            _scopeAttributes = scopeAttributes?.ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
+
+            operationTypeName ??= operation.GetType().Name;
+            _updateStatusScopeName = $"{operationTypeName}.UpdateStatus";
         }
 
         private OperationInternal(OperationState<T> finalState)
-            : base(finalState.RawResponse)
         {
             // FinalOperation represents operation that is in final state and can't be updated.
             // It implements IOperation<T> and throws exception when UpdateStateAsync is called.
             _operation = new FinalOperation();
+            _diagnostics = new ClientDiagnostics(ClientOptions.Default);
```

Comment:
Is it correct to use the default client options here?  If the client was instantiated with sanitizers, shouldn't we be using them for LRO messages?